.gitignore

```diff
@@ -14,5 +14,10 @@ config/honeybadger.yml
 config/settings.local.yml
 config/settings/*.local.yml
 warnings*.log
+/data/
 !/data/.keep
```

Comment:
You could probably delete this file and this line now.

Reply:
Done. Pushing now.